docs/tutorial.rst

```diff
@@ -385,6 +351,57 @@ There may be times when you need to import an existing scratch org that wasn't c
 
 You can hop into a browser logged into any org in your keychain with `cci org browser <org_name>`.
 
+Creating a Connected App
+------------------------
+
+In order to connect persistent orgs such as a Developer Edition, Enterprise Edition, or Sandbox org to CumulusCI, you need to have a Connected App configured in a persistent Salesforce org.
+
+Create With CumulusCI
+^^^^^^^^^^^^^^^^^^^^^
+
+CumulusCI includes a task to easily deploy the Salesforce Connected App to any org in your sdfx keychain.  By default, this will deploy to the org configured as the defaultdevhubusername.
+
+.. code-block:: console
+
+    $ cci task run connected_app
```

Comment:
What about the required `label` option?

Reply:
The default cumulusci.yml sets label to CumulusCI